src/common/MovieTile/index.js

```diff
@@ -0,0 +1,36 @@
+import {
+    Wrapper,
+    Poster,
+    Title,
+    Year,
+    GenreBox,
+    GenreTile,
+    RatingBox,
+    Rating,
+    StarImage,
+    Votes,
+    DescriptionBox
+} from "./styled";
+
+const MovieTile = () => (
+
+    <Wrapper>
+        <Poster />
+        <DescriptionBox>
+            <Title> Mulan </Title>
+            <Year>2020</Year>
+            <GenreBox>
+                <GenreTile>Action</GenreTile>
+                <GenreTile>Drama</GenreTile>
+                <GenreTile>Poema</GenreTile>
+            </GenreBox>
+            <RatingBox>
+                <StarImage></StarImage>
+                <Rating>7,8</Rating>
+                <Votes>35 votes</Votes>
+            </RatingBox>
+        </DescriptionBox>
+    </Wrapper>
+)
```

Comment:
When it comes to a long term usage of this reusable component I think it would've been better to make it more reusable from the start, instead of the way it currently is, because it might require some changes, that we might not wish to spend time later on.

Reply:
Okey, but what to change ?

Reply:
Well instead of typing flat title or Genre here and there it would be better if inside of for example GenreTile would be a prop that'd be coming from MovieTile props, since that's what will happen when we'll use API 💪

Reply:
aah, okey :) Of course ! This version was only for visual checking if everything is just as at Figma :) I can change it in nex commit before merging, no problem :)